sdk/documenttranslation/azure-ai-documenttranslation/azure/ai/documenttranslation/_models.py

```diff
@@ -98,24 +98,29 @@ def _to_generated_list(targets):
 
 class DocumentTranslationInput(object):  # pylint: disable=useless-object-inheritance
     # pylint: disable=C0301
-    """Definition for the input batch translation request.
-
-    :param source_url: Required. Location of the folder / container or single file with your
-     documents.
-    :type source_url: str
-    :param targets: Required. Location of the destination for the output.
-    :type targets: list[TranslationTarget]
-    :keyword str source_language_code: Language code
-     If none is specified, we will perform auto detect on the document.
+    """Input for translation. This requires that you have your source document or
+    documents in an Azure Blob Storage container. Provide a SAS URL to the source file or
+    source container containing the documents for translation. The source document(s) are
+    translated and written to the location provided by the target container SAS URL.
```

Comment:
Target type instead of target container SAS URL

Reply:
This comment is general for everywhere where you mention `target`

Reply:
good catch